### PR TITLE
#159395028 Setup Heroku Deployment For Wger

### PR DESCRIPTION
**Pull request by archbishop12**
#### What does this PR do?
- Sets up Heroku deployment 
#### Description of Task to be completed?
- Create Heroku application instance
- Provision Postgresql database instance
- Link instance to Github project branch
- Set environment variables and enable buildpacks to serve bower components in the project
- Deploy the application to Heroku
#### How should this be manually tested?
Wger staging environment available at this [link](https://wger-aces-staging.herokuapp.com/)

#### Any background context you want to provide?
This is part of the setup to enable continuous delivery of features.

#### What are the relevant pivotal tracker stories?
[#159395028](https://www.pivotaltracker.com/story/show/159395028)

#### Screenshots
screen shot 2018-07-31 at 13 01 17

screen shot 2018-07-30 at 18 15 41

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': '/Users/jameskatarikawe/.local/share/wger/database.sqlite',
```
Comment (isheebo) [inline settings.py]: this name `'/Users/jameskatarikawe/.local/share/wger/database.sqlite',` is local to your system. Pushing it may lead to merge conflicts for other users.

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+NOCAPTCHA = True
+
+# The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
+# This is needed for uploaded files and images (exercise images, etc.) to be
+# properly served.
+SITE_URL = 'http://localhost:8000'
+
+# Path to uploaded files
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+MEDIA_ROOT = '/Users/jameskatarikawe/.local/share/wger/media'
```
Comment (isheebo) [inline settings.py]: same applies to this

File: wger/manager/views/pdf.py [hunk 3]
```diff
@@ -28,7 +28,8 @@
 from wger.utils.pdf import styleSheet
 from wger.utils.pdf import render_footer
 
-from reportlab.lib.pagesizes import A4, cm
+from reportlab.lib.pagesizes import A4
```
Comment (isheebo) [inline wger/manager/views/pdf.py]: I don't think these changes are in the scope of this PR. Perhaps, you may convince me otherwise

File: wger/nutrition/views/plan.py [hunk 4]
```diff
@@ -31,7 +31,8 @@
 from django.views.generic import DeleteView, UpdateView
 
 from reportlab.lib import colors
-from reportlab.lib.pagesizes import A4, cm
+from reportlab.lib.pagesizes import A4
```
Comment (isheebo) [inline wger/nutrition/views/plan.py]: these changes may also not be included in this PR too

File: .gitignore [hunk 5]
```diff
@@ -46,7 +46,6 @@ coverage.xml
 
 # Django stuff:
 *.log
-settings*.py
```
Comment (matthewacha) [inline .gitignore]: Create a PR

Comment (archbishop12) [reply .gitignore]: @matthewacha We added some heroku database configurations to the settings file thus the need to effect this change.
